cmd/utils.go

```diff
@@ -138,13 +129,44 @@ var generateDocsCmd = &cobra.Command{
 	},
 }
 
+// completionCmd generates auto completion commands
+var completionCmd = &cobra.Command{
+	Use:       "completion <shell>",
+	Hidden:    true,
```

Comment:
Should this be hidden?

Reply:
I guess it depends on if we want it visible in the help or not. IMO it's enough to document it in the README and keep it as hidden.

Reply:
If it's a "public" API I think it should be visible